table.go

```diff
@@ -87,11 +88,33 @@ func (t *TableMap) SetUniqueTogether(fieldNames ...string) *TableMap {
 			"gorp: SetUniqueTogether: must provide at least two fieldNames to set uniqueness constraint."))
 	}
 
-	columns := make([]string, 0)
+	columns := make([]string, 0, len(fieldNames))
+	sortedColumns := make([]string, 0, len(fieldNames))
 	for _, name := range fieldNames {
 		columns = append(columns, name)
+		sortedColumns = append(sortedColumns, name)
```

Comment:
I was inclined just to sort `columns`, but that might result in gorp not detecting the previously existing indices since it might pick a different name as such.

Honestly, I think the right medium-term approach is to hand-write the SQL migrations ourselves, including support for +1/-1. It's a bit of overhead, but it's then extremely obvious what changes we're making to the database.